demo/src/main/java/com/eis/geoCalendar/demo/MainActivity.java

```diff
@@ -18,28 +20,39 @@
 import androidx.drawerlayout.widget.DrawerLayout;
 
 import com.eis.geoCalendar.app.network.GenericNetworkEvent;
+import com.eis.geoCalendar.database.EventDatabasePool;
 import com.eis.geoCalendar.demo.Behaviour.EventMapBehaviour;
 import com.eis.geoCalendar.demo.Behaviour.NetworkEventMapBehaviour;
-import com.eis.geoCalendar.demo.Behaviour.OnEventCreatedListener;
-import com.eis.geoCalendar.demo.Behaviour.OnEventRemovedListener;
-import com.eis.geoCalendar.demo.Behaviour.OnEventTriggeredListener;
-import com.eis.geoCalendar.demo.Behaviour.OnMapInitializedListener;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnEventCreatedListener;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnEventRemovedListener;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnEventTriggeredListener;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnMapInitializedListener;
 import com.eis.geoCalendar.demo.Bottomsheet.MapEventBottomSheetBehaviour;
 import com.eis.geoCalendar.demo.Dialogs.AddEventDialog;
 import com.eis.geoCalendar.demo.Dialogs.RemoveEventDialog;
 import com.eis.geoCalendar.demo.Localization.LocationManager;
-import com.eis.geoCalendar.demo.Resources.SMSNetworkEventUser;
 import com.eis.geoCalendar.events.Event;
+import com.eis.geoCalendar.events.EventDatabase;
 import com.eis.geoCalendar.gps.GPSPosition;
 import com.eis.geoCalendar.network.NetworkEvent;
 import com.eis.geoCalendar.network.NetworkEventUser;
+import com.eis.geoCalendar.network.SMS.SMSNetworkEventUser;
+import com.eis.geoCalendar.network.SMS.SMSStringEvent;
 import com.eis.smslibrary.SMSPeer;
 import com.google.android.gms.maps.SupportMapFragment;
 import com.google.android.material.bottomsheet.BottomSheetBehavior;
+import com.google.gson.reflect.TypeToken;
 
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * Current Main activity for a Demo of the geoCalendar that uses an SMS network
+ * <p>
+ * TODO: complete with retrieving of Network events
```

Comment:
This TODO is not nice to see here, but I know this code is still under development, so you can keep it here for the moment.